prov/sm2/src/sm2_common.h

```diff
@@ -53,17 +55,29 @@ extern "C" {
 #endif
 
 #define SM2_VERSION 5
+#define SM2_IOV_LIMIT 4
 
 #define SM2_FLAG_ATOMIC (1 << 0)
 #define SM2_FLAG_DEBUG (1 << 1)
 #define SM2_FLAG_IPC_SOCK (1 << 2)
 #define SM2_FLAG_HMEM_ENABLED (1 << 3)
+#define SM2_CMD_SIZE		256	/* align with 64-byte cache line */
+// reserves 0-255 for defined ops and room for new ops
+// 256 and beyond reserved for ctrl ops
+#define SM2_REMOTE_CQ_DATA (1 << 0)
+#define SM2_TX_COMPLETION (1 << 2)
+#define SM2_RX_COMPLETION (1 << 3)
```

Comment:
A lot of movement of code going on in this patch. It would be much better for this to solely contain the algorithmic changes and for any moving of things to happen before this.

Reply:
fixed.